tests/models/gemma/test_modeling_gemma.py

```diff
@@ -552,8 +552,8 @@ def test_model_2b_eager(self):
         model_id = "google/gemma-2b"
 
         EXPECTED_TEXTS = [
-            "Hello I am doing a project on the 1990s and I need to know what the most popular music",
-            "Hi today I am going to share with you a very easy and simple recipe of <strong><em>Khichdi",
+            "Hello I am doing a lot of research on this topic.\n\nHello,\n\nI have been a member of the ",
+            "Hi today I am going to be honest, I'm not sure if this is a good thing or not",
```

Comment:
Hmm, changing text should have really valid reasons. It's there to keep things from regressing.